state/azure/blobstorage/blobstorage.go

```diff
@@ -149,17 +112,18 @@ func (r *StateStore) Set(req *state.SetRequest) error {
 }
 
 func (r *StateStore) Ping() error {
-	accessConditions := azblob.BlobAccessConditions{}
-
-	if _, err := r.containerURL.GetProperties(context.Background(), accessConditions.LeaseAccessConditions); err != nil {
-		return fmt.Errorf("blob storage: error connecting to Blob storage at %s: %s", r.containerURL.URL().Host, err)
+	getPropertiesOptions := container.GetPropertiesOptions{
```

Comment:
I think this is unnecessary and you should be able to just pass `nil` as 2nd argument for `GetProperties`